src/3_branch.js

```diff
@@ -149,6 +150,19 @@ Branch.prototype._referringLink = function() {
 	else { return null; }
 };
 
+/***
+ * @function Branch._publishEvent
+ * @param {string} event
+ * @param {string} subject
+ */
+Branch.prototype._publishEvent = function(event, subject) {
```

Comment:
I think most event systems just have one concept of event/subject - we can just have people bind to willCloseBanner, didCloseBanner, or both - right?